src/app/layouts/default/lower-sticky-note/lower-sticky-note.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {usePutAway, useStickyData} from '../shared';
+import JITLoad from '~/helpers/jit-load';
+import Cookies from 'js-cookie';
+
+const cookieKey = 'lower-sticky-note-closed';
+
+export default function LowerStickyNote() {
+    const stickyData = useStickyData();
+    const [closed, PutAway] = usePutAway();
+    const shouldNotDisplay =
+        !stickyData ||
+        closed ||
+        stickyData?.mode !== 'banner' ||
+        Boolean(Cookies.get(cookieKey));
+    const hasDisplayed = React.useRef(false);
+
+    React.useEffect(() => {
+        if (hasDisplayed.current && closed) {
```

Comment:
It seems like you could remove hasDisplayed here because closed could only become true from putaway rendering and being interacted with?